RiotSharp/Properties/AssemblyInfo.cs

```diff
@@ -24,4 +24,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.6.0.*")]
+//[assembly: AssemblyVersion("2.6.0")]
```

Comment:
I think it's all in the csproj file now, though the versioning is a bit wonky. As far as I can tell, version must be hardcoded now (no "*") and it's expected that your build tooling overrides one of the composable properties (VersionPrefix and VersionSuffix) to create useful  output. Sadly, I can't give you good reference docs, because I don't know of a way to google for only info about the "new" csproj format.